api/can_api_v2_definition.py

```diff
@@ -129,6 +132,9 @@ class RiskLevels(base_model.APIBaseModel):
     testPositivityRatio: RiskLevel = pydantic.Field(
         ..., description="Test positivity ratio risk level."
     )
+    testPositivityRatioMethod: Optional[
+        can_api_definition.TestPositivityRatioMethod
+    ] = pydantic.Field(None)
```

Comment:
FWIW- I'd advocate we don't add it to the API at all until we have specced out the desired API a bit better.

Reply:
(e.g. as @ghop02 pointed out, "Valorum" isn't what we want to surface in the API)

Reply:
Do we have plans to use it in the short term? I may have been the person to push for this in the API (honestly not 100% sure), but would be good to figure out how to communicate this difference somewhere

Reply:
Actually...  yes, maybe.  I would like to get CMS data in as a backfill this week, and the CMS data is a 14-day average that only has weekly datapoints, and may be up to 7-10 days lagged...  And we probably need to surface that on the site somehow.

I could do that in a hacky way (e.g. hardcode where we're using CMS data or "detect" it based on the cadence of data points)...  or we could do it propely and plumb it through with this provenance work.

The part that makes me nervous is:
* Exposing "Valorum" as a name to API consumers.
* "Valorum" vs "HHSTesting" isn't sufficient, since we also use Covid Tracking Project for ~3 states right now.
* As Chris mentioned, we may want to nest this under a "testPositivityRatioDetails" object so that we give ourselves flexibility to expand.

So it'd be nice if we could expose this as a "private" API until we are happy with it so we don't have to worry about breaking people..  though, in practice probably nobody will notice / use this data, so it may be okay for us to make breaking changes.